frontend/src/e2e-playwright/specs/5_employee/vasu.spec.ts

```diff
@@ -397,5 +397,29 @@ describe('Vasu document page', () => {
         LocalDate.today().format()
       )
     })
+
+    const markDocumentClosed = async () => {
+      const vasuPage = await openDocument()
+      await vasuPage.markClosedButton.click()
+      await vasuPage.modalOkButton.click()
+      await vasuPage.modalOkButton.click()
+    }
```

Comment:
Functions like this could also be moved under `VasuPage`, though the correct abstraction level might be a bit hard to find.